src/gui/models/models.cpp

```diff
@@ -99,9 +99,7 @@ std::function<void()> OCC::Models::addFilterMenuItems(QMenu *menu, const QString
             model->setFilterFixedStringSignalled(filter);
         });
         action->setCheckable(true);
-        if (currentFilter == filter) {
-            action->setChecked(true);
-        }
+        action->setChecked(currentFilter == QRegularExpression::escape(filter));
```

Comment:
The logic change is not explained by the commit message. You could have compared the value to the escaped expression in the `if`, too.

Reply:
Most probably Qt6 porting fallout.
The regex filter api changed for Qt6.

Reply:
It's shorter and does the same thing.

Reply:
> does the same thing

Well, not quite. We have never explicitly set `false` before. It apparently works fine in this case, but I wouldn't assume this as the default in general.

Anyway, good enough for me.